content/user/apps/TrackLibraries.md

```diff
@@ -32,22 +32,36 @@ Export details on findings from the Libraries page, a library's overview page, o
 
 <a href="assets/images/Library-export.png" rel="lightbox" title="Export library details"><img class="thumbnail" src="assets/images/Library-export.png"/></a>
 
-In the Libraries grids, select individual libraries with the check boxes or use your filters to focus on specific data sets. When you apply a selection, you can choose to export all or selected libraries. If you don't select any checkboxes or filters, you can export data for all libraries. 
+In the Libraries grids, select individual libraries or use the filters to focus on specific data sets. If you apply a selection, you can choose to export all or selected libraries; if not, you can export data for all libraries. 
 
 ### Data
 
-The exported file contains the following for each library:
-
-* NAME
-* TECHNOLOGY
-* RELEASE_DATE
-* VERSION
-* LATEST_VERSION
-* HASH
-* CVE <br> (A list of CVEs, separated by commas, found in this library.)
-* NUMBER_APPS <br> (The number of applications that use the library.)
-* NUMBER_CLASSES <br> (The number of classes that use this library.)
-
-
-
+The exported file contains the following data fields for each library:
+
+* Library Name
+* Language
+* Version
+* Release Date
+* Latest Version
+* Grade
+* SHA1
+* CVE Count
+* Application Count
+* Server Count
+* Number of Classes
+* Number of Used Classes
+
+You can access more details on your libraries by going to the [Contrast API documentation](https://api.contrastsecurity.com/#) **> Library > Libraries section**, or using the command line. 
```

Comment:
The hope is that they are getting everything they need from our export, and I don't know how many additional details are actually available in all cases . So maybe we word this a little more subtly?  "Additional library details may be explored using the Contrast RESTful API documentation, or another manual method of your choosing: (then continue with the example below)

Reply:
> You can access more details on your libraries by going to the Contrast API documentation

This language sounds like you can access this data from the documentation, which might technically be true since our API Documentation is interactive and allows users to play with the API in their browser, but I would make the distinction that the data is available from the API, and users can learn about the API from the API Documentation

Reply:
> or using the command line.

I would omit this because it sounds too prescriptive. We can offer an example which happens to use command line tools, but as written it sounds like there is a dedicated Contrast command line tool for access more details about libraries

Reply:
> access more details

Should we specifically call out library associations to Applications and Servers? I believe this is the information that users are looking for a way to export because they want to create reports instead of viewing this data in Contrast

Reply:
I will offer some language for what it's worth:

For users looking to craft custom software composition analysis reports about their applications, the library export feature may not provide sufficient information, but Contrast offers a rich Libraries API for accessing Contrast library data. Reference the [Contrast API documentation](https://api.contrastsecurity.com/#) **> Library > Libraries section** for instructions on using the Libraries API.